db/migrate/20170525135108_create_bookable_slots.rb

```diff
@@ -0,0 +1,12 @@
+class CreateBookableSlots < ActiveRecord::Migration[5.1]
+  def change
+    create_table :bookable_slots do |t|
+      t.belongs_to :schedule, null: false, foreign_key: true, index: true
+      t.date :date, null: false
+      t.string :start, null: false
+      t.string :end, null: false
```

Comment:
The naming of this has caused one or two issues given it's a reserved word in Ruby. I've worked around it by qualifying the references and to maintain consistency with the slots generated by the `DefaultBookableSlots` generator but am thinking of renaming instead...